Configuration/AlCa/python/autoCond.py

```diff
@@ -42,9 +42,9 @@
     # GlobalTag for Run3 data relvals (prompt GT) - identical to 123X_dataRun3_Prompt_v10 but with snapshot at 2022-05-31 20:00:00 (UTC)
     'run3_data_prompt'             : '123X_dataRun3_Prompt_frozen_v4',
     # GlobalTag for Run3 offline data reprocessing
-    'run3_data'                    : '123X_dataRun3_v7',
+    'run3_data'                    : '123X_dataRun3_Candidate_2022_06_09_08_51_52',
```

Comment:
@cms-sw/alca-l2 there are quite a bit of spurious changes here because the Queue is not as updated as the GT in autoCond:

```
$ conddb diff 123X_dataRun3_v7 123X_dataRun3_Candidate_2022_06_09_08_51_52
[2022-06-09 10:53:12,203] INFO: Connecting to pro [frontier://PromptProd/cms_conditions]
Record                  Label                                          pro::123X_dataRun3_v7 Tag                                           pro::123X_dataRun3_Candidate_2022_06_09_08_51_52 Tag          
----------------------  ---------------------------------------------  ------------------------------------------------------------------  -----------------------------------------------------------   
AlCaRecoTriggerBitsRcd  -                                              AlCaRecoHLTpaths_2021_offline_v1                                    AlCaRecoHLTpaths_2021_offline_v2                              
GBRDWrapperRcd          electron_eb_ECALonly                           GEDelectron_EBCorrection_80X_EGM_v4                                 GEDelectron_EBCorrection_ForOffline_v1                        
GBRDWrapperRcd          electron_eb_ECALonly_var                       GEDelectron_EBUncertainty_80X_EGM_v4                                GEDelectron_EBUncertainty_ForOffline_v1                       
GBRDWrapperRcd          electron_eb_ecalOnly_1To300_0p0002To0p5_sigma  electron_eb_ecalOnly_1To300_0p0002To0p5_sigma_RunIToUL2016To2018V1  electron_eb_ecalOnly_1To300_0p0002To0p5_sigma_ForOffline_v1   
GBRDWrapperRcd          electron_eb_ecalOnly_1To300_0p2To2_mean        electron_eb_ecalOnly_1To300_0p2To2_mean_RunIToUL2016To2018V1        electron_eb_ecalOnly_1To300_0p2To2_mean_ForOffline_v1         
GBRDWrapperRcd          electron_eb_ecalTrk_1To300_0p0002To0p5_sigma   electron_eb_ecalTrk_1To300_0p0002To0p5_sigma_RunIToUL2016To2018V1   electron_eb_ecalTrk_1To300_0p0002To0p5_sigma_ForOffline_v1    
GBRDWrapperRcd          electron_eb_ecalTrk_1To300_0p2To2_mean         electron_eb_ecalTrk_1To300_0p2To2_mean_RunIToUL2016To2018V1         electron_eb_ecalTrk_1To300_0p2To2_mean_ForOffline_v1          
GBRDWrapperRcd          electron_ee_ECALonly                           GEDelectron_EECorrection_80X_EGM_v4                                 GEDelectron_EECorrection_ForOffline_v1                        
GBRDWrapperRcd          electron_ee_ECALonly_var                       GEDelectron_EEUncertainty_80X_EGM_v4                                GEDelectron_EEUncertainty_ForOffline_v1                       
GBRDWrapperRcd          electron_ee_ecalOnly_1To300_0p0002To0p5_sigma  electron_ee_ecalOnly_1To300_0p0002To0p5_sigma_RunIToUL2016To2018V1  electron_ee_ecalOnly_1To300_0p0002To0p5_sigma_ForOffline_v1   
GBRDWrapperRcd          electron_ee_ecalOnly_1To300_0p2To2_mean        electron_ee_ecalOnly_1To300_0p2To2_mean_RunIToUL2016To2018V1        electron_ee_ecalOnly_1To300_0p2To2_mean_ForOffline_v1         
GBRDWrapperRcd          electron_ee_ecalTrk_1To300_0p0002To0p5_sigma   electron_ee_ecalTrk_1To300_0p0002To0p5_sigma_RunIToUL2016To2018V1   electron_ee_ecalTrk_1To300_0p0002To0p5_sigma_ForOffline_v1    
GBRDWrapperRcd          electron_ee_ecalTrk_1To300_0p2To2_mean         electron_ee_ecalTrk_1To300_0p2To2_mean_RunIToUL2016To2018V1         electron_ee_ecalTrk_1To300_0p2To2_mean_ForOffline_v1          
GBRDWrapperRcd          pfscecal_EBCorrection_offline_v2               pfscecal_EBCorrection_offline_v2_RunIToUL2016To2018V1               pfscecal_EBCorrection_ForOffline_v1                           
GBRDWrapperRcd          pfscecal_EBUncertainty_offline_v2              pfscecal_EBUncertainty_offline_v2_RunIToUL2016To2018V1              pfscecal_EBUncertainty_ForOffline_v1                          
GBRDWrapperRcd          pfscecal_EECorrection_offline_v2               pfscecal_EECorrection_offline_v2_RunIToUL2016To2018V1               pfscecal_EECorrection_ForOffline_v1                           
GBRDWrapperRcd          pfscecal_EEUncertainty_offline_v2              pfscecal_EEUncertainty_offline_v2_RunIToUL2016To2018V1              pfscecal_EEUncertainty_ForOffline_v1                          
GBRDWrapperRcd          photon_eb_ECALonly                             GEDphoton_EBCorrection_80X_EGM_v4                                   GEDphoton_EBCorrection_ForOffline_v1                          
GBRDWrapperRcd          photon_eb_ECALonly_var                         GEDphoton_EBUncertainty_80X_EGM_v4                                  GEDphoton_EBUncertainty_ForOffline_v1                         
GBRDWrapperRcd          photon_eb_ecalOnly_5To300_0p0002To0p5_sigma    photon_eb_ecalOnly_5To300_0p0002To0p5_sigma_RunIToUL2016To2018V1    photon_eb_ecalOnly_5To300_0p0002To0p5_sigma_ForOffline_v1     
GBRDWrapperRcd          photon_eb_ecalOnly_5To300_0p2To2_mean          photon_eb_ecalOnly_5To300_0p2To2_mean_RunIToUL2016To2018V1          photon_eb_ecalOnly_5To300_0p2To2_mean_ForOffline_v1           
GBRDWrapperRcd          photon_ee_ECALonly                             GEDphoton_EECorrection_80X_EGM_v4                                   GEDphoton_EECorrection_ForOffline_v1                          
GBRDWrapperRcd          photon_ee_ECALonly_var                         GEDphoton_EEUncertainty_80X_EGM_v4                                  GEDphoton_EEUncertainty_ForOffline_v1                         
GBRDWrapperRcd          photon_ee_ecalOnly_5To300_0p0002To0p5_sigma    photon_ee_ecalOnly_5To300_0p0002To0p5_sigma_RunIToUL2016To2018V1    photon_ee_ecalOnly_5To300_0p0002To0p5_sigma_ForOffline_v1     
GBRDWrapperRcd          photon_ee_ecalOnly_5To300_0p2To2_mean          photon_ee_ecalOnly_5To300_0p2To2_mean_RunIToUL2016To2018V1          photon_ee_ecalOnly_5To300_0p2To2_mean_ForOffline_v1     
```

we might just test this now as is (and accept the changes in the comparisons) or wait for update of GTs, which is needed anyway and then test.

Reply:
Addressed at [ae11863](https://github.com/cms-sw/cmssw/pull/38297/commits/ae11863599543d1ef25823d751ca5a04c9bee970) .